### ISSUE TITLE
Outgoing Connector

### ISSUE THREAD

**Issue by TwinMist** [sebbmeyer/php-microsoft-teams-connector#8]
Hi
Are you doing any connector for outgoing?

thanks

**Comment by sebbmeyer** [comment]
@TwinMist do you mean this https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/outgoingwebhook ?
What exactly are you looking for?

**Comment by TwinMist** [comment]
Yes, do you have anything writen in PHP for this to wait and get the notification
thanks

**Comment by sebbmeyer** [comment]
No, nothing is planned for this yet. But I'm happy to merge it, if anybody submits a pull request for it
